### DISCUSSION TITLE
how to run deploy.ts and tasks files in the project.

### DISCUSSION THREAD

**Discussion by shakeib98** [PaulRBerg/hardhat-template#185] [Q&A] [answered]
In the package.json file there is no script defined for it. 
And running individually the files of tasks giving Property 'ethers' does not exist on type 'HardhatRuntimeEnvironment'.

I used npx ts-node ./tasks/accounts.ts

**Comment by PaulRBerg** [comment, marked as answer]
Hey @shakeib98, thanks for using my template!

This repo is not meant to be a replacement for Hardhat - instead, it is an aggregator of various utilities that are known to be needed in most Solidity projects.

To run the `accounts` task, you can do:

```sh
$ yarn hardhat accounts
```

But that is just an example. Refer to the official documentation on [how to create tasks][1] and [how to write scripts][2] with Hardhat.

[1]: https://hardhat.org/guides/create-task.html
[2]: https://hardhat.org/guides/scripts.html
